homeassistant/components/nordpool/strings.json

```diff
@@ -12,12 +12,20 @@
         "data": {
           "currency": "Currency",
           "areas": "Areas"
+        },
+        "data_description": {
+          "currency": "Select currency to display prices in, EUR is the base currency.",
```

Comment:
base? Do we convert it? I think `default currency` sounds more logical

Reply:
It's nordpools base currency. All other are converted by them from EUR hence why we also have a sensor to show the fx rate.

So it's not a default, it's the base currency.